internal/dbutil/pgutil/data.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package pgutil
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/internal/dbutil/dbschema"
+)
+
+// QueryData loads all data from tables
+func QueryData(db Queryer, schema *dbschema.Schema) (*dbschema.Data, error) {
```

Comment:
hmm everything except `quote_nullable` look to be independent from `postgres`. Maybe it can moved to `dbutil`?

Reply:
Hmm, I could move it into `dbschema with an optional `quoteColumn func(string)` ?

Reply:
yes, I think thats easiest solution.

Reply:
done